eng/helix/GetNodejsVersion.ps1

```diff
@@ -0,0 +1,31 @@
+[CmdletBinding(PositionalBinding=$false)]
+Param(
+    [int] $MajorVersion,
```

Comment:
```suggestion
        [Parameter(Mandatory=$true)][int] $MajorVersion,
```
Let's validate that the required parameters are actually specified in each of these scripts.